### PR TITLE
Create an overlay for no accounts

### PR DESCRIPTION
**Pull request by jacogr**
Closes https://github.com/polkadot-js/apps/issues/1101

Polkadot:Substrate Portal 2019-05-08 11-45-21

### CODE REVIEW

**Comment by jacogr** [comment]
Don't think this is the prettiest thing ever by any stretch of the imagination.

File: packages/apps/src/overlays/Accounts.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    const { isApiReady } = this.props;
+    const { isDismissed, hasAccounts } = this.state;
+
+    if (!isApiReady || isDismissed || hasAccounts) {
+      return null;
+    }
+
+    return (
+      <Wrapper>
+        <Trans i18nKey='noAccounts'>You have no accounts. Some features such as the ability to send extrinsics only become available once you have accounts. <Link to ='/accounts'>Create an account now.</Link></Trans>
```
Comment (Tbaut) [inline packages/apps/src/overlays/Accounts.tsx]: If you ask me, I wouldn't talk about `extrinsic` and rather be general, along the line:
```suggestion
        <Trans i18nKey='noAccounts'>You have no accounts. Most features require an account to work properly. <Link to ='/accounts'>Create an account now.</Link></Trans>
```

Comment (jacogr) [reply packages/apps/src/overlays/Accounts.tsx]: The issue is that the stuff doesn't show, so some adjustment is probably needed, but your suggestion gives less information. Maybe we should explicitly say that "some features are hidden"

**Comment by Tbaut** [comment]
:+1:

**Pull request merged**